bundles/security/index.rst

```diff
@@ -106,6 +106,50 @@ the object type and id are also passed the permissions of the security contexts
 from the role might be overridden by the permissions from this specific object
 (which are handled by the previously mentioned ``AccessControlManager``).
 
+Single-Sign-On  Authentication
+------------------------------
+
+Sulu supports authentication via Single-Sign-On (SSO).
+To enable it, the security configuration needs to be adjusted to allow SSO in the admin firewall.
+This can be configured in the ``config/packages/security.yaml``:
+
+.. code-block:: diff
+
+    security:
+        # ...
+
+        firewalls:
+            # ...
+            admin:
+
+                # ...
+                logout:
+                    path: sulu_admin.logout
+   +            access_token:
+   +                token_handler: sulu_security.single_sign_on_token_handler
+   +                token_extractors: sulu_security.single_sign_on_token_extractor
+
+    # ...
+    sulu_security:
+        checker:
+            enabled: true
+        password_policy:
+            enabled: true
+   +    single_sign_on:
+   +        providers:
+   +            'sulu.io':
+   +                dsn: 'openid://%env(resolve:SULU_OPEN_ID_CLIENT_ID)%:%env(resolve:SULU_OPEN_ID_CLIENT_SECRET)%@%env(resolve:SULU_OPEN_ID_ENDPOINT)%'
+   +                user_role: 'USER'
```

Comment:
While write the blog post I'm asking if we should name this `role` or `role_key`. The user role entity is a different one 🤔  /cc @chirimoya @Prokyonn

Reply:
`default_role`, `default_role_key`?

Reply:
I think `default_role_key` is the most accurate, I'd go with that